app/lib/server/methods/addUsersToRoom.js

```diff
@@ -30,7 +30,7 @@ Meteor.methods({
 		const userInRoom = subscription != null;
 
 		// Can't add to direct room ever
-		if (room.t === 'd') { // TODO CHANGE
+		if (room.t === 'd') {
```

Comment:
Why this one isn't a check against the enum?

Reply:
just because I have a feeling we gonna change this very soon (direct messages between multiple users)